README.md

```diff
@@ -0,0 +1,51 @@
+# Vanna Javascript (Browser) Client
+
+# Introduction
+
+Vanna is an internal feature flagging service used at [PBS](http://pbs.org). It
+helps deliver new features to users quickly and safely. This is an example
+client implementation for browser based Javascript.
+
+To learn more about the rationale behind using feature flags, read these
+articles:
+
+- https://martinfowler.com/articles/feature-toggles.html
+- https://blog.travis-ci.com/2014-03-04-use-feature-flags-to-ship-changes-with-confidence/
+
+## Install
+
+The recommended way to install vanna is through `npm`.
+
+```sh
+npm install @pbs/vanna
+```
+
+## Setup
+
+After installing `vanna`, you can import it with your Javascript bundler of
+choice and setup the client.
+
+```js
+import vanna from "@pbs/vanna";
+
+const client = vanna.setup({
+  server: "https://vanna.services.pbs.org/",
+  userSegment: "beta-tester"
+});
+
+client.on("ready", () => {
+  const isFeatureEnabled = client.variation("your-feature-slug", {
+    fallback: false
```

Comment:
I think `default` here might be clearer than `fallback`

Reply:
`default` is a [reserved word in JS](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/default). That'll lead to problems with code like:

```javascript
const options = { default: 'hi' };
const { default } = options;
```

It'll blow up with: `default is a reserved word`

Naming things is still hard 😃